.github/workflows/secret-scanning.yml

```diff
@@ -0,0 +1,78 @@
+name: Secret Scanning
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
this is not available when run in a fork – I don't think anyone in a fork can add a secret we would care about, so we might want to skip this pipeline completely for fork PRs (or rewrite it, so this permission is not needed)